<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SellerTorrentPluginDialog</class>
 <widget class="QDialog" name="SellerTorrentPluginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>869</width>
    <height>248</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Seller extension</string>
  </property>
  <widget class="QTabWidget" name="tabs">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>849</width>
     <height>229</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="peers">
    <attribute name="title">
     <string>Peers</string>
    </attribute>
    <widget class="QTreeView" name="peerPluginsTableView">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>16</y>
       <width>819</width>
       <height>167</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="settings">
    <attribute name="title">
     <string>Settings</string>
    </attribute>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>148</y>
       <width>87</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>154</y>
       <width>287</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Changes will only apply to future contracts</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="maxNumberOfSellersLineEdit">
     <property name="geometry">
      <rect>
       <x>196</x>
       <y>42</y>
       <width>57</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="minPriceLineEdit">
     <property name="geometry">
      <rect>
       <x>196</x>
       <y>14</y>
       <width>57</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QTimeEdit" name="maxConfirmationTimeTimeEdit">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>44</y>
       <width>55</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="time">
      <time>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
      </time>
     </property>
    </widget>
    <widget class="QLabel" name="minFeeLabel">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>72</y>
       <width>139</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Min. fee/kB (m…É):</string>
     </property>
    </widget>
    <widget class="QLabel" name="minLockLabel">
     <property name="geometry">
      <rect>
       <x>278</x>
       <y>16</y>
       <width>129</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Min. Lock time (h:m):</string>
     </property>
    </widget>
    <widget class="QLabel" name="maxNumberOfSellersLabel">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>44</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Maximum #sellers:</string>
     </property>
    </widget>
    <widget class="QLabel" name="maxConfirmationTimeLabel">
     <property name="geometry">
      <rect>
       <x>278</x>
       <y>46</y>
       <width>167</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Max. conf. time (h:m):</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="minFeeLineEdit">
     <property name="geometry">
      <rect>
       <x>196</x>
       <y>70</y>
       <width>57</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QTimeEdit" name="minLockTimeEdit">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>12</y>
       <width>57</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="minPriceLabel">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>16</y>
       <width>143</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Price/piece (satoshies):</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
